ts/components/staking/wizard/wizard_flow.tsx

```diff
@@ -638,7 +638,7 @@ const StartStaking: React.FC<StartStakingProps> = props => {
                               `You're delegating ${stakingAmountTotalComputed} ZRX to ${
                                   selectedStakingPools.length > 1
                                       ? `${selectedStakingPools.length} pools`
-                                      : `${selectedStakingPools[0].pool.metaData.name || '1 pool'}`
+                                      : `${utils.getPoolDisplayName(selectedStakingPools[0].pool)}`
```

Comment:
Here I was a bit lazy but I think `Pool 1` works as well but open to changing it into a custom truncation here